CONTRIBUTING.md

```diff
@@ -113,6 +111,89 @@ You can use the che command `dashboard_dev_server` (command pallette > Run task
 $ node_modules/.bin/gulp serve --server=<che_api_url>
 ```
 
+### Workspace Loader
+Workspace loader source code is located in [<this.repo>/workspace-loader](./workspace-loader/) folder.
+Here is the developer workflow if you want to contribute to it:
+
+#### Devfile for workspace loader development
+
+The devfile: [<this.repo>/workspace-loader/devfile.yaml](./workspace-loader/devfile.yaml)
+
+In this section, we show how to setup a Che 7 environment to work on the che workspace loader, and how to use it.
+For the whole workflows, we will need a workspace with such containers:
+
+- Workspace Loader Dev container (a.k.a ws-loader-dev) is a all in one container for running commands such as build, test or start the workspace loader server.
+
+All containers have `/projects` folder mounted, which is shared among them.
+
+Developer workflow:
+
+1. Start the workspace with the devfile, it is cloning Che repo.
+2. Install dependencies
+3. Build
+4. Code ...
+5. Run unit test
+6. Start workspace loader server and preview
```

Comment:
To me `and test` sounds better.

Reply:
I do not think so because we already have the test command.